projects/packages/action-bar/composer.json

```
@@ -0,0 +1,61 @@
+{
+	"name": "automattic/jetpack-action-bar",
+	"description": "An easy way for visitors to follow, like, and comment on your site.",
+	"type": "jetpack-library",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"automattic/jetpack-assets": "1.17.x-dev",
+		"automattic/jetpack-constants": "1.6.x-dev"
+	},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "1.0.3",
+		"automattic/jetpack-changelogger": "^3.2",
+		"automattic/wordbless": "dev-master"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"phpunit": [
+			"./vendor/phpunit/phpunit/phpunit --colors=always"
+		],
+		"test-coverage": [
+			"php -dpcov.directory=. ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\""
+		],
+		"test-php": [
+			"@composer phpunit"
+		],
+		"build-production": "echo 'Add your build step to composer.json, please!'",
+		"build-development": "echo 'Add your build step to composer.json, please!'",
```

You should be able to add your build steps here now.

```suggestion
		"build-production": [
			"pnpm run build-production"
		],
		"build-development": [
			"pnpm run build"
		],
```